tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -412,7 +412,7 @@ CGPathRef newPathForRoundRect(CGRect rect, CGFloat radius) {
     return path;
 }
 
-TEST(CGPath, CGPathContainsPointOutsideRect) {
+DISABLED_TEST(CGPath, CGPathContainsPointOutsideRect) {
```

Comment:
>DISABLED_TEST [](start = 0, length = 13)

These tests should have been disabled initially since they involve work with arcs/curves. These will be re-enabled then.